core/server/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -185,7 +186,7 @@ private boolean tooManyPendingPackets() {
   }
 
   /**
-   * Requires to hold mLock when calling this.
+   * Requires to hold mLock when calling this. This is only called in the event loop thread.
```

Comment:
for "Requires to hold mLock when calling this", should we just use the following annotation instead:
```java
 @GuardedBy("mLock")
```